include/zephyr/pm/policy.h

```diff
@@ -211,7 +210,7 @@ void pm_policy_device_power_lock_put(const struct device *dev);
  * If an event is registred, it will return the number of ticks until the next event as
  * a positive or zero value. Otherwise it returns -1
```

Comment:
Could you clarify that if there is an unregistered past event, this function returns zero?

Reply:
Updated to:

```
/**
 * @brief Returns the ticks until the next event
 *
 * If an event is registred, it will return the number of ticks until the next event, if the
 * "next"/"oldest" registered event is in the past, it will return 0. Otherwise it returns -1.
 *
 * @retval >0 If next registered event is in the future
 * @retval 0 If next registered event is now or in the past
 * @retval -1 Otherwise
 */
 ```